package.json

```diff
@@ -58,19 +58,19 @@
     }
   },
   "dependencies": {
-    "@tensorflow/tfjs": "^1.5.1",
     "long": "^4.0.0",
     "syft-proto": "git+https://github.com/OpenMined/syft-proto.git",
     "tuple-w": "^1.1.1"
   },
   "peerDependencies": {
-    "@tensorflow/tfjs": "^1.2.5"
+    "@tensorflow/tfjs-core": "^1.5.1"
```

Comment:
Lower the version here.  This is the minimum version we allow to be used in our library.  Unless something has changed, I know 1.2.5 worked when I last tried it.  I'm sure we can go even further back to 1.0.0 at some point.  We should accept greater than 1.2.5 at the very least.

Reply:
Lowered